build/depends.py

```diff
@@ -1236,6 +1236,8 @@ def sources(self, build):
                    "util/indexrange.cpp",
                    "util/widgetrendertimer.cpp",
 
+                   "idcmp/junction.cpp",
```

Comment:
You should find a reasonable folder name for your feature, related to the feature itself.